src/hooks/useTokens.ts

```diff
@@ -1,67 +1,76 @@
-import { ChainType, getTokens } from '@lifi/sdk';
-import type { TokensResponse } from '@lifi/sdk';
-import { useQuery } from '@tanstack/react-query';
 import {
-  getTokenBySymbol as getTokenBySymbolHelper,
-  getTokenByName as getTokenByNameHelper,
-  getTokenByAddressOnSpecificChain as getTokenByAddressOnSpecificChainHelper,
   getNativeTokenForChain as getNativeTokenForChainHelper,
+  getTokenByAddressOnSpecificChain as getTokenByAddressOnSpecificChainHelper,
+  getTokenByName as getTokenByNameHelper,
+  getTokenBySymbol as getTokenBySymbolHelper,
 } from '@/utils/tokenAndChain';
+import type { TokensResponse } from '@lifi/sdk';
+import { ChainType, getTokens } from '@lifi/sdk';
+import { useQuery } from '@tanstack/react-query';
+import { useCallback, useMemo } from 'react';
 
 export const queryKey = ['tokenStats'];
 
 export const getTokensQuery = async () => {
   const tokens = await getTokens({
     chainTypes: [ChainType.EVM, ChainType.SVM, ChainType.UTXO, ChainType.MVM],
   });
+  console.log(
+    'tokens',
+    tokens.tokens[1].filter((token) => token.address.startsWith('0x00')),
+  );
   return tokens;
 };
```

Comment:
Please remove console.log
```suggestion
  return tokens;
};
```

Reply:
Arr, nice catch 🙏